modules/contact-form/grunion-contact-form.php

```diff
@@ -1370,6 +1370,9 @@ static function _style_on() {
 	 * @return string HTML for the concat form.
 	 */
 	static function parse( $attributes, $content ) {
+		if ( function_exists( 'get_current_screen' ) && get_current_screen()->id === 'sync' ) {
```

Comment:
I think we might be better off setting a constant during sync, like `DOING_SYNC`